src/components/pages/books/BookItem.js

```diff
@@ -0,0 +1,11 @@
+import BooksDetails from './BookDetails';
+
+function BookItem() {
+  return (
+    <li className="booksItem">
+      <BooksDetails />
+    </li>
+  );
```

Comment:
- [ ] This is a good start but instead of calling back the component, please use the Router.